infra/README.md

```diff
@@ -1,12 +1,103 @@
+# Ecommerce web app deployed on Kubernetes
+
+This GitHub repository contains the Terraform and Kubernetes YAML used by the Jump Start Solution (JSS) entitled [_Ecommerce web app deployed on Kubernetes_](https://cloud.google.com/products/solutions/details/ecomm-microservices), available on Google Cloud.
```

Comment:
```suggestion
This GitHub repository contains the Terraform and Kubernetes YAML used by the Jump Start Solution (JSS) entitled [_Ecommerce web app deployed on Kubernetes_](https://console.cloud.google.com/products/solutions/details/ecomm-microservices), available on Google Cloud.
```
Sorry, it looks like I made a typo. Feel free to exclude this fix in your PR. I'm happy to fix the typo myself in a diff PR.